lib/puppet/type/cisco_vrf.rb

```diff
@@ -81,6 +81,32 @@ def self.title_patterns
     end
   end # property description
 
+  newproperty(:mhost_ipv4) do
```

Comment:
Please add these new properties to the doc example at the beginning of this file.